.github/workflows/ci.yml

```diff
@@ -34,6 +34,12 @@ jobs:
         with:
           source-folder: code
 
+      # Run build tasks
+      - name: Run default build tool task
+        uses: matlab-actions/run-build@v1
```

Comment:
Lets use `@v2` for all of the matlab-actions now. We should also update the setup-matlab and commented run-matlab examples